src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Calculator {
+    public double calculate(double firstDigit, double secondDigit, char operator) {
+        switch (operator) {
+            case '+':
+                return firstDigit + secondDigit;
+            case '-':
+                return firstDigit - secondDigit;
+            case '*':
+                return firstDigit * secondDigit;
+            case '/':
+                if (secondDigit == 0) {
+                    throw new ArithmeticException("Dividing by zero forbidden");
+                } else {
+                    return firstDigit / secondDigit;
+                }
```

Comment:
do we need else after `throw`?
```suggestion
                return firstDigit / secondDigit;
```